dotorg/dotorg-topic-highlighter.user.js

```diff
@@ -158,17 +158,70 @@ jQuery(document).ready(function( $ ) {
  		$( '#tamper-wp-topic-highlighter-nonport' ).prop( 'checked', settings.nonPOrT );
 	});
 
+
     if ( $( 'body' ).is( '.bbp-view.archive' ) ) {
         $( '#bbpress-forums .bbp-pagination:first' ).after( `
-        <div class="custom-topic-header">
-            <label for="tec-select-all">
-                <input id="tec-select-all" type="checkbox">
-                Select all Topics
-            </label>
-            <input type="submit" id="tec-open-in-new-tab" value="Open" />
-        </div>
+			<div class="custom-topic-header">
+				<label for="tec-select-all">
+					<input id="tec-select-all" type="checkbox">
+					Select all Topics
+				</label>
+				<input type="submit" id="tec-open-in-new-tab" value="Open" />
+			</div>
+        ` );
+
+        var totalOnPageTicket = $( '.topic' ).length;
+        var totalNotLastVoiceTicket = $(".topic:not(.tamper-last-voice)").length;
+        var totalLoggedInTicket = $(".tamper-logged-in-ticket").length;
+        var totalNewTicket = $(".tamper-new-ticket").length;
+        var totalToResolveTicket = $(".tamper-to-resolve").length;
+        var totalFollUpTicket = Math.abs(totalNotLastVoiceTicket - totalNewTicket);
+
+		var paginationContainer = $( '.bbp-pagination' );
+
+        // Show stats beside the pagination to and bottom
```

Comment:
I don't understand this comment

Reply:
@Camwyn another typo should be "top and bottom"